specs/introduction.md

```diff
@@ -60,13 +60,13 @@ In order to scale Ethereum without sacrificing security, we must preserve 3 crit
 liveness, availability, and validity.
 
 1. **Liveness** - Anyone must be able to extend the rollup chain by sending transactions at any time.
```

Comment:
Slight nit here: numbered lists in markdown should always use `1` and the markdown rendering client will automatically increment them, this is to reduce diffs when making changes

Reply:
then i think we can use this opportunity to change all the numbers here to `1`?